spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -499,4 +499,44 @@ describe("Router", function()
         check(i, args[3], args[4]))
     end
   end)
+
+  describe("url encoded uris", function()
+
+    setup(function()
+      helpers.dao:truncate_tables()
+      assert(helpers.dao.apis:insert {
+        name         = "localbin",
+        upstream_url = "http://localhost:9999/",
+        uris         = {
+          "/",
+        },
+        hosts        = {
+          "localbin.com",
+        },
+      })
+
+      assert(helpers.start_kong {
+        nginx_conf = "spec/fixtures/custom_nginx.template",
```

Comment:
I checked all the `httpbin.org` use that we have (maybe I missed something), but we use basically these:

```
/get
/post
/response-headers
/status/200
/delay
```

This already covers something like 90% of our use cases:
https://github.com/Mashape/kong/pull/2315/files#diff-cf7ff3021b29019f806a6f4de24e493d

And the missing pieces can be added really easily.

I don't agree that we should keep using `httpbin.org` when we already have the most of the functionality that we need in those 40 lines of code, and I would say that it even exceeds `httpbin.org` in some of its functionality already (I do have tests that are not even possible with `httpbin.org`).

It would take me about a day to move all our `httpbin.org` tests to that localbin, if that decision is made. In my opinion, we cal do everything with one or two endpoints, and we don't need and should not replicate `httpbin.org`.